__model/Reviews.model.js

```diff
@@ -26,11 +26,38 @@ exports.SelectReviews = (order = "DESC" ) => {
       });
   };
   
-  exports.SelectReviewById = (review_id) => {
+  exports.SelectReviewById = (rev_id) => {
     return db
       .query("SELECT * FROM reviews WHERE review_id = $1;", [review_id])
       .then((result) => {
         if(result.rows.length === 0){
           return Promise.reject({ status: 404, msg: "ID not found !" });}
         return result.rows[0]});
+  }
+  exports.patchComment = (rev_id,votes) => {
+    const {inc_votes} = votes
+    return checkIDExists(rev_id).then(() => {
```

Comment:
You dont actually need to do this here, you can do the update query with RETURNING * and see how many rows were altered with the review id that you give, if result.rows.length === 0 then you know that the review didnt exist, allowing you to do 1 query instead of 2